src/Core/Content/Seo/Api/SeoActionController.php

```diff
@@ -155,6 +155,10 @@ public function updateCanonicalUrl(RequestDataBag $seoUrl, Context $context): Re
             throw SeoException::salesChannelNotFound($salesChannelId);
         }
 
+        if ($salesChannel->getTypeId() === Defaults::SALES_CHANNEL_TYPE_API) {
```

Comment:
not sure if I should throw an exception here or skip it. or not adjust the controller here at all.